src/main/java/de/tum/in/www1/artemis/service/ParticipationService.java

```diff
@@ -388,7 +392,13 @@ public ProgrammingExerciseStudentParticipation resumeExercise(Exercise exercise,
             return null;
         }
         participation = copyBuildPlan(participation);
-        participation = configureBuildPlan(participation);
+        continuousIntegrationService.get().enablePlan(participation.getBuildPlanId());
```

Comment:
I don't like these changes, because they break the transaction mechanism that we have implemented here. Please make sure that the logic in configureBuildPlan is followed.